internal/store/fleetdb/components.go

```diff
@@ -1014,9 +1013,8 @@ func (r *Store) setAttributes(component *fleetdbapi.ServerComponent, attr *attri
 			}).Warn("error in conversion of versioned attributes to raw data")
 	}
 
-	// skip min sized json data containing just the braces `{}`
-	smallest := 2
-	if len(data) == smallest {
+	// skip min sized json data containing just the braces `{}`.
+	if len(data) == len("{}") {
```

Comment:
this is just as fast due to len() being computed at compile time, but also more clear on what we are doing.